homeassistant/components/slack/notify.py

```diff
@@ -117,29 +164,42 @@ def _async_templatize_blocks(hass, value):
             key: _async_templatize_blocks(hass, item) for key, item in value.items()
         }
 
-    tmpl = template.Template(value, hass=hass)
+    tmpl = template.Template(value, hass=hass)  # type: ignore
```

Comment:
Why is this annotation needed?

Reply:
Since I added the `slack` integration to the list of strictly typed once (in `setup.cfg`), I get the `Call to untyped function "Template" in typed context` error without it.

I'll add an additional comment with the mypy error code.